CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 11.3.4
+
+This release of Teleport contains a low severity security fix and multiple improvements and bug fixes.
+
+### (Low) Arbitrary length data reading in go-mssqldb
+
+* Fixed issue with malicious MS SQL server causing potential DoS with malformed client packet. [go-mssqldb#7] https://github.com/gravitational/go-mssqldb/pull/7, [#21639](https://github.com/gravitational/teleport/pull/21639)
```

Comment:
I just want to confirm what I think this is saying, that it's the sever that's malicious?

Reply:
Yep.